src/settingsmanager.cpp

```diff
@@ -175,6 +176,11 @@ void SettingsManager::initSettings()
     QVariant defaultLangVariant(defaultLangList);
     m_langList = m_settings.value("language", defaultLangVariant).toList();
 
-    m_categoryList = m_settings.value("category", {}).toList();
+    const auto lang = QLocale().name().split('_')[0].toStdString();
+    auto categories = deducePair(m_settings.value("category", {}).toList());
+    for (auto& category : categories) {
+        category.first = QString::fromStdString(kiwix::translateBookCategory(lang, category.second.toStdString()));
+    }
+    setCategory(categories);
```

Comment:
IMHO, a better/simpler solution would be *not* saving the translations of the categories in the settings/session file. But that would mean incompatibility with old session files.

Reply:
Same opinion. I planned to open an issue on this after merge. It has too many side effects for the scope of this issue.

Reply:
@ShaopengLin If something has to be fixed later, then please open an issue immediatly and link. Don't wait the PR is merged.

Reply:
Moved to #1185